### ISSUE TITLE
1.8 NEI causing failure, no crash

### ISSUE THREAD

**Issue by TherminatorX** [Chicken-Bones/NotEnoughItems#217]
Using NotEnoughItems-1.8-1.0.5.104-universal and CodeChickenCore-1.8-1.0.5.36-universal only, NEI fails right after trying to be initialized.

Forge log: http://pastebin.com/dXpzYVZD

If it's a ID-ten-T error, I apologize.

**Comment by GirafiStudios** [comment]
First of all, update your Forge. A lot of stuff have changed/been fixed since the version you're using.

**Comment by Chicken-Bones** [comment]
Somehow FML has managed to give you a log, without the stack trace (useful error information), try with the latest versions, and tell me what happens

**Comment by TherminatorX** [comment]
W/o mods, the new Forge (1.8-8.0.81.1079) runs just fine. Adding CCC (same version as above), however, causes the following crash.
http://pastebin.com/YTrp49hD

(I know the issue is not NEI here, but bear with me. That's next.)

**Comment by TherminatorX** [comment]
Alright, back on track. Using Forge 8.99.81.1423 now. (Stupid Win installer)
Everything looks fairly similar, and the process is the same: W/ just CCC everything's nice, as soon as you add NEI and fire it up, BOOM! 
![capture2](https://cloud.githubusercontent.com/assets/7256249/7945048/dfb3925c-0933-11e5-9cad-a57f610e792b.JPG)
http://pastebin.com/hU0QduTL

**Comment by GirafiStudios** [comment]
What version of Java are you using?

**Comment by TherminatorX** [comment]
J8U25, but updating to 8-45 doesn't fix the issue.

**Comment by TherminatorX** [comment]
New Forge 8.99.88.1427 (odd, the profile says I'm using Forge 11.14.2.1427) still has the issue.

**Comment by TherminatorX** [comment]
Nothing new, @Chicken-Bones?

**Comment by PawnedSauce** [comment]
I am having the same problem I have tried both the Latest 1.8 build as well as the Recommended of both CCC and NEI, neither seems to work and I have also tried to downgrade Forge versions, no luck.

 I get some problem with `[FML]: Model definition for location storagedrawers:compDrawers#inventory not found` the other mod that I have installed, as seen, is Storage Drawers. I think this might be a problem of FML's side - likely?

Used Forge ranging from 11.14.2.30-39
Using the recommended of Forge is too outdated

**Comment by TherminatorX** [comment]
@PawnedSauce Nah, that shows up normally w/o either of CB's mods installed. I have CustomNPCs installed, and that throws up a similar warning with no ill effect.

**Comment by PawnedSauce** [comment]
@TherminatorX But it could be my issue that I am receiving is that Forge is breaking things along the way, just like the way that it is trying to find models of that certain object and throwing a fit about it. Could be but that is my conclusion :/
